releasenotes/notes/52519.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: traffic-management
+issue:
+  - 52519
+
+releaseNotes:
```

Comment:
We should add ENABLE_ENHANCED_DESTINATIONRULE_MERGE=false to the compatibility versions so users using those will get the old behavior.